FrontEnd/README.md

```diff
@@ -1,6 +1,6 @@
 # FrontEnd
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 17.3.2.
+This was generated with [Angular CLI](https://github.com/angular/angular-cli) version 17.3.2.
```

Comment:
Comment

Reply:
It is better for me to keep the word project

Reply:
Pues yape